### ISSUE TITLE
`TypeVarTuple` cannot have value constraints

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#6901]
Pyright fails the following test in the newly-written type system conformance test suite. It does not generate an error in the following case.

```python
Ts2 = TypeVarTuple("Ts2", int, float)  # Type error
```

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.345, which I just published. It will also be included in a future release of pylance.
